openedx/core/djangoapps/notifications/utils.py

```diff
@@ -47,6 +47,10 @@ def get_show_notifications_tray(user):
     return show_notifications_tray
 
 
+def get_is_new_notification_view_enabled():
+    return ENABLE_NEW_NOTIFICATION_VIEW.is_enabled()
```

Comment:
Flag is implemented as course waffle fiag.  Do we need course waffle flag type? if yes why was course_id not passed here?

Reply:
We need this flag to be enabled for everyone, if course_id is not provided the CourseWaffleFlag checks for permission (everyone, admin etc) and enables it accordingly

Reply:
I think it is better to use `WaffleFlag` here as we don't need to enable it for a single course